PC/winreg.c

```diff
@@ -1605,13 +1605,11 @@ winreg_SetValue_impl(PyObject *module, HKEY key, const Py_UNICODE *sub_key,
     long rc;
 
     if (type != REG_SZ) {
-        PyErr_SetString(PyExc_TypeError,
-                        "Type must be winreg.REG_SZ");
+        PyErr_SetString(PyExc_TypeError, "type must be winreg.REG_SZ");
         return NULL;
     }
-    if (value_length >= INT_MAX) {
-        PyErr_SetString(PyExc_OverflowError,
-                        "the value is too long");
+    if (value_length >= PY_DWORD_MAX) {
```

Comment:
```suggestion
    if ((size_t)value_length >= PY_DWORD_MAX) {
```
to avoid implicit cast to signed or unsigned... i never recall if the compiler prefers signed or unsigned :-p